pkg/iso9660util/iso9660util.go

```diff
@@ -33,10 +31,6 @@ func Write(isoPath, label string, layout []Entry) error {
 	if err != nil {
 		return err
 	}
-	if runtime.GOOS == "windows" {
-		// go-embed unfortunately needs unix path
-		workdir = filepath.ToSlash(workdir)
-	}
```

Comment:
Is this covered by current windows tests?

Reply:
This path is not used in WSL2 setup - it doesn't load ISO image, but instead an archive is used. So, no, it is not covered, because QEMU support on Windows is still WIP

Reply:
It is only created when you run `start`